docs/abi.md

```diff
@@ -62,6 +63,10 @@ outgoing messages over the write halves of channels. The `oak_main` function is
 generally expected to run forever, but may return a status if the Node choses to
 terminate (whether expectedly or unexpectedly).
 
+Each Oak Application starts with a single initial Oak Node; this Node receives
```

Comment:
I don't think so, because one of the changes in this PR is that there is no longer any externally-visible concept of a node name.  It's still there, but it's entirely internal and just used for diagnostics &ndash; the running Application can't see it, and the `ApplicationConfiguration` has no mention of it.